### PR TITLE
Fixes grammar, punctuation, brevity, and clarity

### PR DESCRIPTION
**Pull request by SKhubaib**
## Why

Line 12: tutorial: missing article - change to 'a tutorial'.
Line 12: quick-reference: missing article and unnecessary hyphen - change to 'a quick reference'.
Line 14: a number of: appears twice, and is against your guidelines for brevity - change to 'many' and 'several'.
Line 20: in order to: against your guidelines for brevity - change to 'to'.
Line 20: and version-control it as part of your application: incomplete idea - change to 'and version-control it as part of your application files'.
Line 26: Git,: incorrect punctuation in a compound predicate - remove the comma after Git.
Line 26: setup: verb misspelled twice as a noun - change to 'set up'.
Line 
Line 28: environment-complete: missing space before and after the &mdash - add space before and after the &mdash.
Line 29: or to automated CI systems: redundant preposition - remove 'to'.
Line 40: etc...: extra periods after etc. - change to 'etc.', removing the two extra periods.
Line 41: Redis etc.: missing punctuation - change to 'Redis, etc.', adding comma after Redis.
Line 41: etc..: extra period after etc. - change to 'etc.', removing the extra period.
Line 42: minute-including and data-to create.: missing spaces before and after the &mdash - add spaces before and after each &mdash.
Line 44: updated and secure: contradicting forms of verb - change to 'up-to-date and secure'.

## What's changed

Grammar, punctuation, spelling, brevity, and clarity

### CODE REVIEW

File: docs/src/_index.md [hunk 1]
```diff
@@ -9,24 +9,24 @@ If you're new to Platform.sh, we recommend starting with the **Big Picture**, in
...
 
-You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as is any database, search engine, or cache that you specify.
+You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git and push.  You don't need to set up anything manually. The web server is already set up and configured, as is any database, search engine, or cache that you specify.
 
-Every branch you push can be made a fully independent environment&mdash;complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
-Its automatically generated URL can be sent to stakeholders or to automated CI systems.
+Every branch you push can be made a fully independent environment &mdash; complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
```
Comment (CollierCZ) [inline docs/src/_index.md:28]: :clipboard: 
There shouldn't be spaces around em dashes: https://developers.google.com/style/dashes?hl=en#em-dashes

```suggestion
Every branch you push can be made a fully independent environment&mdash;complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything.
```

File: docs/src/_index.md [hunk 2]
```diff
@@ -37,8 +37,8 @@ Platform.sh respects the structure of branches. It's entirely up to you.
...
-3. **Full Cluster Cloning Technology:** The full production cluster can be cloned in under a minute&mdash;including all of its data&mdash;to create on-the-fly, ephemeral development environments that are a byte-level copy of production.
+1. **Unified Environment:** All of your services (MySQL, ElasticSearch, MongoDB, etc.) are managed inside the cluster and included in the price, with no external single-points-of-failure. When you back up an environment, you get a fully consistent snapshot of your whole application.
+2. **Multi-Services & Multi-App:** You can deploy multiple applications (for example, in a microservice-based architecture), using multiple data backends (MySQL, PostgreSQL, Redis, etc.) written in multiple frameworks (Drupal + NodeJS + Flask, for example) in multiple languages, all in the same cluster.
+3. **Full Cluster Cloning Technology:** The full production cluster can be cloned in under a minute &mdash; including all of its data &mdash; to create on-the-fly, ephemeral development environments that are a byte-level copy of production.
```
Comment (CollierCZ) [inline docs/src/_index.md:42]:
```suggestion
3. **Full Cluster Cloning Technology:** The full production cluster can be cloned in under a minute&mdash;including all of its data&mdash;to create on-the-fly, ephemeral development environments that are a byte-level copy of production.
```

**Comment by SKhubaib** [comment]
Can the instances of 'etc.' also be removed from this document? I think there were at least 2 occurrences.

**Comment by SKhubaib** [comment]
> Can the instances of 'etc.' also be removed from this document? I think there were at least 2 occurrences.

As discussed in the other Pull Request, there should be no Latin terms, abbreviated or not.

**Comment by SKhubaib** [comment]
> They certainly can be removed. As I mentioned, this page has a lot of things that don't fit the style guide.

Also the first sentence is ambiguous. Second generation Platform as a service? What are the other generations? I couldn't find any definitions around that. Thanks

**Review by CollierCZ — APPROVED** [review]
Thanks for the changes! We'll definitely try to improve the wording in a future iteration.

**Pull request merged**